htdocs/core/class/html.form.class.php

```diff
@@ -2344,13 +2344,13 @@ function select_produits_fournisseurs_list($socid,$selected='',$htmlname='produc
      *  @param      string	$htmlname        Name of HTML field
      *  @return		string|null
      */
-    function select_product_fourn_price($productid,$htmlname='productfournpriceid')
+    function select_product_fourn_price($productid, $selected_supplier='', $htmlname='productfournpriceid')
```

Comment:
Sure, but we this method is also a good candidate to be used by eternal modules, so we must keep compatibility when we can. Breaking is don something when probability to see function used by external module is lower.